Tasks/AppStoreRelease/app-store-release.ts

```diff
@@ -287,7 +292,11 @@ async function run() {
             } else {
                 let bundleIdentifier: string = tl.getInput('appIdentifier', false);
                 pilotCommand.arg(['pilot', 'upload', ...authArgs]);
-                pilotCommand.arg(['-i', filePath]);
+                if (applicationType.toLocaleLowerCase() === 'macos') {
+                    pilotCommand.arg(['-P', filePath]);
```

Comment:
Should we apply the same for 'Production' track ('deliver' command)?

Reply:
fastlane deliver is already called correctly (using -c for pkg and -i for ipa)